### PR TITLE
Adds IPMenu v1.1

### PR DESCRIPTION
**Pull request by bweave**
I think I've done everything right to create the new Cask. It downloaded properly to my machine and passed the audit. Thanks for taking a look!

Description: IPMenu is a free utility that conveniently puts both your local and Internet IP address into your menu bar. To see your IP addresses, simply click on the icon. IPMenu can also e-mail you when your Internet IP address changes.

### CODE REVIEW

File: Casks/ipmenu.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Ipmenu < Cask
```
Comment (vitorgalvao) [inline Casks/ipmenu.rb]: You’re using the old scheme. It changed quite a while back. This line should now be `cask :v1 => 'ipmenu' do`

File: Casks/ipmenu.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  version :latest
+  sha256 :no_check
+
+  url 'http://www.macupdate.com/download/14221/ipmenu.zip'
+  homepage 'http://www.macupdate.com/app/mac/14221/ipmenu'
+  license :unknown
```
Comment (vitorgalvao) [inline Casks/ipmenu.rb]: `:gratis`.

**Comment by vitorgalvao** [comment]
Should we send this to [caskroom/unofficial](https://github.com/caskroom/homebrew-unofficial), @rolandwalker? All macupdate links.

File: Casks/ipmenu.rb (hunk 1 above)
Comment (radeksimko) [inline Casks/ipmenu.rb]: The truth is that we should be checking this in `audit` so such casks don't pass the audit.

Comment (vitorgalvao) [inline Casks/ipmenu.rb]: We should. It’s being caught by Travis, though, so that helps.

**Comment by vitorgalvao** [comment]
Closing. @bweave If you’d like this to still be included, please submit it to [caskroom/unofficial](https://github.com/caskroom/homebrew-unofficial) with the requested changes.

**Pull request closed without merging**